sda-commons-web-autoconfigure/src/test/java/org/sdase/commons/spring/boot/web/metadata/MetadataOtherServiceAuthenticatedClient.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2022- SDA SE Open Industry Solutions (https://www.sda.se)
+ *
+ * Use of this source code is governed by an MIT-style
+ * license that can be found in the LICENSE file or at
+ * https://opensource.org/licenses/MIT.
+ */
+package org.sdase.commons.spring.boot.web.metadata;
+
+import org.sdase.commons.spring.boot.web.client.AuthenticationPassThroughClientConfiguration;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.http.HttpHeaders;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@FeignClient(
+    value = "metadataOtherAuthenticated",
+    url = "${metadata.otherAuthenticated.baseUrl}",
+    configuration = {AuthenticationPassThroughClientConfiguration.class})
+public interface MetadataOtherServiceAuthenticatedClient {
```

Comment:
Is the metadata context added unconditionally to all clients? Imho that should be only used for clients that are calling with the platform. Like the authentication pass through is configured here with `AuthenticationPassThroughClientConfiguration`. That should also be documented somewhere, like in `EnableSdaClients` or [the docs](https://github.com/SDA-SE/sda-spring-boot-commons/blob/b5aa5acb20a9e454c3ebcb718846f3e2c97bb825/docs/web/index.md#http-client) (which are actually missing the trace token config).